packages/web-runtime/tests/unit/components/Account/CalDavUrl.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { defaultComponentMocks, defaultPlugins, mount } from '@opencloud-eu/web-test-helpers'
+import { flushPromises } from '@vue/test-utils'
+import { nextTick } from 'vue'
+import CalDavUrl from '../../../../src/components/Account/CalDavUrl.vue'
+import { vi } from 'vitest'
+
+import { ref } from 'vue'
+
+beforeEach(() => {
+  vi.stubGlobal('navigator', {
+    clipboard: {
+      writeText: vi.fn()
+    }
+  })
+})
+
+vi.mock('@opencloud-eu/web-pkg', async (importOriginal) => {
+  const actual = await importOriginal()
+  return {
+    ...(actual as Record<string, unknown>),
+    useClientService: () => ({
+      httpAuthenticated: {
+        get: vi.fn().mockResolvedValue({
+          status: 301,
+          headers: {
+            location: '/caldav/'
+          },
+          request: {
+            responseURL: 'https://example.com/caldav/'
+          }
+        })
+      }
+    }),
+    useUserStore: () => ({
+      user: ref({
+        onPremisesSamAccountName: 'test-user'
+      })
+    }),
+    useConfigStore: () => ({
+      serverUrl: 'https://example.com/'
+    })
+  }
+})
+
+describe('CalDavUrl component', () => {
+  it('renders CalDAV information when available', async () => {
+    const wrapper = mount(CalDavUrl, {
+      global: {
+        plugins: defaultPlugins(),
+        mocks: {
+          ...defaultComponentMocks,
+          $gettext: (msg: string) => msg,
+          authStore: {
+            userContextReady: true
+          }
+        }
+      },
+      props: {
+        checked: true,
+        calDavAvailable: true
+      }
+    })
```

Comment:
I'd suggest having one method per test file that returns the wrapper. See e.g. how it's done the unit tests for the app tokens: https://github.com/opencloud-eu/web/blob/main/packages/web-runtime/tests/unit/components/Account/AppTokens.spec.ts